app/src/app/poll/poll.component.ts

```diff
@@ -119,6 +119,24 @@ export class PollComponent implements OnInit, OnDestroy {
   currentRoundBallot : PollBallot = NONE_BALLOT;
   justVoteBallot     : PollBallot = NONE_BALLOT;
 
+  //movement of sidebar
+  openSide(){
+    var sidedom:any=document.getElementById('info-panel');
+    var content:any=document.getElementById('information');
+    var closebtn:any=document.getElementById('close-button');
+    sidedom.style.transform="translate(0,0)";
+    content.style.display="contents";
+    closebtn.style.display="block";
+  }
+  closeSide(){
+    var sidedom:any=document.getElementById('info-panel');
+    var content:any=document.getElementById('information');
+    var closebtn:any=document.getElementById('close-button');
+    sidedom.style.transform="translate(90%,0)";
+    content.style.display="none";
+    closebtn.style.display="none";
+  }
+  
```

Comment:
To test you implementation, you can set the classes of the information panel to `class="info-panel closed"` and change it to `class="info-panel opened"` manually in the HTML part of the inspector (then back to `closed`). This will force you to implement all the animation in SASS only, and it will be as close as possible to the final implementation.